### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by sdmeijer** [primo-io/prototype-documentation#20]
I've found some extra images and addes some to the Dutch translation. Maybe they can also be added to the other translations, it clarifies some of the text.

**Comment by mat-lo** [comment]
yes, that's a good idea, we can add them to the other pages as well.

**Comment by sdmeijer** [comment]
can you resize them as well, they're a bit large.
